src/xpk/commands/batch.py

```diff
@@ -46,11 +48,66 @@ def batch(args: Namespace) -> None:
 
 def submit_job(args: Namespace) -> None:
   cmd = (
-      'kubectl-kjob create slurm --profile'
-      f' {APP_PROFILE_TEMPLATE_DEFAULT_NAME} --'
-      f' {args.script}'
+      'kubectl kjob create slurm'
+      f' --profile {APP_PROFILE_TEMPLATE_DEFAULT_NAME}'
+      f' --localqueue {LOCAL_QUEUE_NAME}'
   )
 
+  if args.ignore_unknown_flags:
+    cmd += ' --ignore-unknown-flags'
+  if args.first_node_ip:
+    cmd += ' --first-node-ip'
+
+  cmd += f' -- {args.script}'
+
+  if args.array is not None:
+    cmd += f' --array {args.array}'
+
+  if args.cpus_per_task is not None:
+    cmd += f' --cpus-per-task {args.cpus_per_task}'
+
+  if args.gpus_per_task is not None:
+    cmd += f' --gpus-per-task {args.gpus_per_task}'
+
+  if args.mem is not None:
+    cmd += f' --mem {args.mem}'
+
+  if args.mem_per_task is not None:
+    cmd += f' --mem-per-task {args.mem_per_task}'
+
+  if args.mem_per_cpu is not None:
+    cmd += f' --mem-per-cpu {args.mem_per_cpu}'
+
+  if args.mem_per_gpu is not None:
+    cmd += f' --mem-per-gpu {args.mem_per_gpu}'
+
+  if args.nodes is not None:
```

Comment:
On kjob we don't have constraints for this flag.

Reply:
What max number of nodes we should set? WDYT?

Reply:
Any thoughts?

Reply:
Based on the command used to create clusters on GKE, I'd say that we can set 1000 as the max limit:

https://github.com/AI-Hypercomputer/xpk/blob/c1551023d58599e3cbae76e559ebae6a23c7aa82/src/xpk/commands/cluster.py#L479

Reply:
Ah. Not sure. This is nodes. But nodes on the Slurm it is pods.

Reply:
Then, probably we can use 110? It seems that's the default max num of pods per node on a GKE cluster. Documentation: https://cloud.google.com/kubernetes-engine/docs/how-to/flexible-pod-cidr#max_pods_default

Looks like it's possible to change that value, but I don´t see we're changing it in xpk.

Reply:
Or based on the info here https://kubernetes.io/docs/setup/best-practices/cluster-large/, should we set a limit of 150K? @pawloch00 WDYT?

Reply:
Ok, let's leave it unconstrained, If I understand https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.25/#job-v1-batch correctly, it will be checked.